InnerEye/ML/config.py

```diff
@@ -518,6 +518,16 @@ def len_or_zero(lst: Optional[List[Any]]) -> int:
         if self.mean_teacher_alpha is not None:
             raise ValueError("Mean teacher model is currently only supported for ScalarModels."
                              "Please reset mean_teacher_alpha to None.")
+        if not self.disable_extra_postprocessing:
+            if self.slice_exclusion_rules is not None:
+                for rule in self.slice_exclusion_rules:
+                    if rule.higher_class not in self.ground_truth_ids:
```

Comment:
this would be better placed in an instance method "validate", similar to SummedProbRule